ui/src/index.tsx

```diff
@@ -20,13 +24,21 @@ export const MyContext = React.createContext<IAppContext>(null);
 const AppProvider = (props) => {
   const [store, setStore] = useState({
     volume: null,
+    version: "",
```

Comment:
To be sure what we're referring to:

```suggestion
    sdkVersion: "",
```